addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -98,12 +98,11 @@ function factory(dependencies) {
             if (!chatWindow) {
                 chatWindow = this.env.models['mail.chat_window'].create({
                     manager: [['link', this]],
-                    threadViewer: [['create', {
-                        thread: [['link', thread]],
-                    }]],
+                    thread: [['link', thread]],
                 });
             }
-            if (mode === 'last_visible' && !chatWindow.isVisible) {
+            // FIXME should use isVisible but isVisible is considered true too soon
+            if (mode === 'last_visible' &&  !chatWindow.visibleIndex !== undefined) {
```

Comment:
I see FIXME for using isVisible, but in the meantime this is more clear to read
```suggestion
            if (mode === 'last_visible' &&  chatWindow.visibleIndex === undefined) {
```